pandas/tests/plotting/test_datetimelike.py

```diff
@@ -303,7 +302,7 @@ def test_uhf(self):
         tlocs = axis.get_ticklocs()
         tlabels = axis.get_ticklabels()
         for loc, label in zip(tlocs, tlabels):
-            xp = conv._from_ordinal(loc).strftime("%H:%M:%S.%f")
+            xp = conv._from_ordinal(loc).strftime("%H:%M:%S")
```

Comment:
Confirmed the decimal here is not shown in the label when plotting.

Reply:
should it be?

Reply:
Great question. On matplotlib 3.5, the code `rs = str(label.get_text())` returns the empty string and the subsequent assert is not ran. On 3.6, we get `"21:59:51"` for `rs`. In both cases, `21:59:51` is displayed on the resulting plot. So I believe the test was incorrect on our end.